web-client/src/app/state/session.service.ts

```diff
@@ -58,6 +60,15 @@ export class SessionService {
       if ('Created' in result) {
         const wallet = result.Created;
         this.sessionStore.update({ wallet, pin });
+        try {
+          await this.searchService.insertWalletAddress({
+            wallet_id: wallet.wallet_id,
+            phone_number: wallet.phone_number,
+            owner_name: wallet.owner_name,
+          });
+        } catch (err) {
+          console.log(err);
```

Comment:
I believe this should be `console.error`

Reply:
No, there are other places in our code that has console.log(err) - https://github.com/ntls-io/nautilus-wallet/blob/23a328a827c853028880e7222da5a5487246cbbf/web-client/src/app/views/register/register.page.ts#L109

Reply:
I see.  I won't pretend to understand it but I assume it relates to how actionable the error message is.